tile-join.cpp

```diff
@@ -659,7 +659,7 @@ struct tileset_reader {
 				}
 			}
 		} else {
-			overzoomed_tiles = next_overzoomed_tiles;
+			overzoomed_tiles = std::move(next_overzoomed_tiles);
 			next_overzoomed_tiles.clear();
```

Comment:
nit: I think the clear() call is unnecessary now... but also shouldn't cause any harm